android/modules/ui/src/java/ti/modules/titanium/ui/widget/TiUILabel.java

```diff
@@ -135,6 +135,7 @@ public boolean onTouchEvent(MotionEvent event) {
 		tv.setKeyListener(null);
 		tv.setFocusable(false);
 		tv.setSingleLine(false);
+		tv.setEllipsize(ellipsize);
```

Comment:
How does that effect current labels that do not have any `ellipsize` property defined? We should try to ensure that nothing breaks after this change (although 6.0 would be the right time to do this).